platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -61,7 +61,11 @@
           "org.codice.ddf.registry.federationadmin.service.impl.RefreshRegistryEntries",
           "org.codice.ddf.registry.policy.RegistryPolicyPlugin",
           "Registry_Configuration_Event_Handler",
-          "org.codice.ddf.registry.api.impl.RegistryStoreCleanupHandler");
+          "org.codice.ddf.registry.api.impl.RegistryStoreCleanupHandler",
+          "ddf.catalog.federation.impl.CachingFederationStrategy",
+          "ddf.catalog.metacard.validation.MetacardValidityFilterPlugin",
```

Comment:
❗️ As of DDF 2.13.6, there is a config file in the etc directory that configures this service.
https://github.com/codice/ddf/commit/4e5ccf69e0c120603d466e4c039890209f4f0d87
You should test to verify, but I believe doing import from a DDF 2.13.3-2.13.5 will end up overwriting what's in the etc directory and misconfiguring the system. We need to make sure the field looks like this after import and restart:
`invalid-state\=<DDF-HOSTNAME>-data-manager......`
instead of this:
`invalid-state\=data-manager......`.
Let me know if you need help testing this or help fixing it, if my assumption is correct.